support-frontend/assets/components/checkoutNudge/checkoutNudgeContainer.tsx

```diff
@@ -28,15 +28,24 @@ export function CheckoutNudgeContainer({
 	const [displayNudge, setDisplayNudge] = useState(true);
 
 	const recurringType =
-		abParticipations.singleToRecurringV2 === 'control' ||
-		!abParticipations.singleToRecurringV2
+		abParticipations.singleToRecurringV3 === 'control' ||
+		!abParticipations.singleToRecurringV3
 			? 'MONTHLY'
 			: 'ANNUAL';
 	const currencyGlyph = glyph(detect(countryGroupId));
 	const minAmount = config[countryGroupId][recurringType].min;
-	const paragraphCopyVariant = `Regular, reliable support powers Guardian journalism in perpetuity. If you can, please consider setting up ${
-		recurringType === 'MONTHLY' ? 'a' : 'an'
-	} ${recurringType.toLowerCase()} payment today from just  ${currencyGlyph}${minAmount} – it takes less than a minute.`;
+
+	const minWeeklyAmount =
+		countryGroupId === 'GBPCountries'
+			? Math.ceil((minAmount * 100) / 52).toString() + `p`
+			: currencyGlyph +
+			  (Math.ceil((minAmount * 100) / 52) / 100).toFixed(2).toString();
+
+	const title2CopyMonthly = `Support us every month`;
+	const paragraphCopyMonthly = `Regular, reliable support powers Guardian journalism in perpetuity. If you can, please consider setting up a ${recurringType.toLowerCase()} payment today from just  ${currencyGlyph}${minAmount} – it takes less than a minute.`;
+
+	const title2CopyAnnual = `From just ${currencyGlyph}${minAmount} (${minWeeklyAmount} a week)`;
+	const paragraphCopyAnnual = `Funding Guardian journalism every year doesn’t need to be expensive. Make a bigger impact today, and protect our independence long term. Please consider annual support.`;
```

Comment:
This is a bit confusing, it's quite difficult to tell from the code what the resultant string will be. It would probably be best to use a function here as determining the string is more complicated. You could also use a switch statement inside the function

Reply:
Ah yes, have placed in a neater function (line 50) so easier to read I hope.

Reply:
That function looks great! Just one small note on the switch statement